packages/Sandblocks-Core/SBBlock.class.st

```diff
@@ -1221,6 +1262,12 @@ SBBlock >> isArtefact [
 	^ false
 ]
 
+{ #category : #'as yet unclassified' }
+SBBlock >> isAttractedTo: aMorph [
+	
+	^ aMorph class == self class
```

Comment:
Here, an isKindOf: could possibly be even nicer? It would change the meaning to "if the other object's class is a specialization of my class", which may or may not be the intent.